src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -356,6 +358,19 @@ private static Job CreateJobForGivenRuntime(Job baseJob, string runtimeId, Comma
                     builder.TargetFrameworkMoniker(runtime.MsBuildMoniker);
 
                     return baseJob.WithRuntime(runtime).WithToolchain(builder.ToToolchain());
+                case RuntimeMoniker.Wasm:
```

Comment:
ideally this wouldn't be needed and we could just use `RuntimeMoniker.NetCoreApp50` with a `runtimeId=browser-wasm`

Reply:
@adamsitnik Is that actually a reasonable thing to do? There doesn't seem to be any where else where this is conditions on runtimeID, and I am not sure how easy it would be to get the right information in the right place?